src/server.js

```diff
@@ -33,27 +32,15 @@ module.exports = function() {
 (node ${colors.green(process.versions.node)} on ${colors.green(process.platform)} ${process.arch})`);
 	log.info(`Configuration file: ${colors.green(Helper.CONFIG_PATH)}`);
 
-	if (!fs.existsSync("public/js/bundle.js")) {
-		log.error(`The client application was not built. Run ${colors.bold("NODE_ENV=production npm run build")} to resolve this.`);
-		process.exit();
-	}
-
 	var app = express()
+		.disable("x-powered-by")
```

Comment:
Seriously Express...

Unrelated to this PR, but it might be worth looking into https://expressjs.com/en/advanced/best-practice-security.html#use-helmet. What do you think?

Reply:
It doesn't provide any advantage.